project/apps/account/src/app/strategies/jwt.strategy.ts

```diff
@@ -10,7 +10,7 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
     super({
       jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
       ignoreExpiration: false,
-      secretOrKey: configService.get('JWT_SECRET') || 'secret-jwt-key',
+      secretOrKey: configService.get('JWT_SECRET') as string,
```

Comment:
Функция вроде дженерик
```suggestion
      secretOrKey: configService.get<string>('JWT_SECRET'),
```